verb.el

```diff
@@ -909,12 +909,12 @@ This function is called from ob-verb.el (`org-babel-execute:verb')."
       ;; re-reading the request in the current level (contained in the
       ;; source block). If no more levels exist, skip the call to
       ;; `verb--request-spec-from-hierarchy'.
-      (when (verb--up-heading)
+      (if (verb--up-heading)
         ;; Continue reading requests from the headings
         ;; hierarchy. Pre-include the one we read from the source block
         ;; at the end of the list.
-        (setq rs (verb--request-spec-from-hierarchy rs)))
-      (verb--request-spec-post-process rs))))
+        (setq rs (verb--request-spec-from-hierarchy rs))
```

Comment:
Since we're skipping the call to `verb--request-spec-post-process` in this `if` branch, then the call to `setq` is no longer needed. We can replace this line with just the call to `verb--request-spec-from-hierarchy`.

Reply:
Yes indeed. Thanks for pointing it.